modules/references/client/components/read-references/Reference.js

```diff
@@ -65,6 +84,9 @@ export default function Reference({ reference }) {
               {t('Member since {{memberSince}}', {
                 memberSince: new Date(userFrom.created).getFullYear(),
               })}
+              {getGender(userFrom.gender) === undefined
```

Comment:
Shorthand would work nicely here:
```
{userFrom.gender && getGender(...)}
```

Reply:
I tried this but I'm not quite sure what you mean... we checked to see if the gender is undefined because if it is not we wanted to leave the comma out. Do you mean we should solve this differently?

Reply:
I meant that `userFrom.gender` will already be undefined without passing it through function, and its enough to check for it being falsy.

Since the alternative ternary output is just an empty string, I think I'd do something like this:

```
{{userFrom.gender && `${getGender(userFrom.gender)}, `}}
```

Returning undefined in JSX will just render nothing.

Does that make sense?

It's not a biggie but reads nicer to me. :-)

Reply:
I meant that `userFrom.gender` will already be undefined without passing it through function, and its enough to check for it being falsy.

Since the alternative ternary output is just an empty string, I think I'd do something like this:

```
{{userFrom.gender && `${getGender(userFrom.gender)}, `}}
```

Returning undefined in JSX will just render nothing.

Does that make sense?

It's not a biggie but reads nicer to me. :-)

Reply:
it reads a lot nicer! thanks, I'm learning loads!